conda/install.py

```diff
@@ -138,6 +138,13 @@ def _remove_readonly(func, path, excinfo):
     os.chmod(path, stat.S_IWRITE)
     func(path)
 
+def warn_failed_remove(function, path, exc_info):
+    if exc_info[1].errno == errno.EACCES:
+        log.warn( "WARNING: cannot remove, permission denied: %s" % path )
```

Comment:
With log.warn, I don't think you need to have `WARNING` in the string.  The logger will tag the message as a warning when it outputs to stdout.